lite/examples/model_personalization/README.md

```diff
@@ -142,7 +142,7 @@ directory.
 
 The more flexible option is to prepare the models you want to
 use as base and head as TensorFlow SavedModels. If you are using Keras,
-`tf.keras.experimental.export_saved_model` can be used to do this.
```

Comment:
Worth saying:

`tf.keras.experimental.export_saved_model` can be used to do this in TF1, and in TF2 you can use `tf.keras.models.save_model`.

Reply:
@jdduke I've added a part about 1.x - I live a bit in the future with 2.x only :D